spec/controllers/api/type_scholarships_controller_spec.rb

```diff
@@ -57,4 +57,117 @@
       end
     end
   end
+  describe 'PATCH update' do
+    context 'when user is signed in' do
+      let(:user) { FactoryBot.create(:user) }
+      
+      subject do
+        request.headers['Authorization'] = "Bearer #{generate_token(user)}"
+        patch :update, params: params
+
+        response
+      end
+
+      context 'with valid data' do
+        context 'with scholarship=:bidding and description' do
```

Comment:
Cambiar el texto por `when scholarship is bidding`. El `and description` no sé qué significa.
Tené en cuanta que estos textos son para que sean legibles por humanos. Tiene que tener sentido la frase en idioma natural